modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -1116,6 +1116,17 @@ void ONNXImporter::populateNet(Net dstNet)
                 layerParams.set("zoom_factor_y", scales.getIntValue(2));
                 layerParams.set("zoom_factor_x", scales.getIntValue(3));
             }
+            else if(node_proto.input_size() == 2 && constBlobs.find(node_proto.input(1)) == constBlobs.end())
```

Comment:
This way i think it's not critical if second input will be non-constant. That way it works as "resize first input as second".

Reply:
yeah, you are right. Second input need not to be non-constant. I have changed it. Please have a look now.